platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationReportImpl.java

```diff
@@ -161,49 +164,64 @@ public MigrationReport getReport() {
 
   @SuppressWarnings(
       "PMD.DefaultPackage" /* designed to be called from ExportMigrationEntryImpl within this package */)
+  ExportMigrationReportImpl recordFile(ExportMigrationEntryImpl entry) {
+    SecurityUtils.doPrivileged(
```

Comment:
❓ I can't see why this would need to run privileged.

Reply:
😄 Copy and paste and I was originally going to add the file permissions attribute as well. I'll take it out.